GVFS/GVFS.Notifications/VFSForGit.Mac/StatusMenuItem/UI/VFSAboutWindowController.m

```diff
@@ -0,0 +1,58 @@
+#import "VFSAboutWindowController.h"
+
+@interface VFSAboutWindowController ()
+
+@property (strong) VFSProductInfoFetcher *productInfoFetcher;
+
+@end
+
+@implementation VFSAboutWindowController
+
+- (instancetype _Nullable)initWithProductInfoFetcher:(VFSProductInfoFetcher *_Nonnull)productInfoFetcher
+{
+    if (self = [super initWithWindowNibName:@"VFSAboutWindowController"])
+    {
+        if (!productInfoFetcher)
```

Comment:
Check input parameter before super init instead of going through init and later returning nil anyway. Also, compare pointers to nil instead of treating them as bools.